### DISCUSSION TITLE
Release v2.0.0

### DISCUSSION THREAD

**Discussion by roseduan** [lotusdblabs/lotusdb#117] [Announcements]
## 🚀 New Features
* Change the architecture of LotusDB
  * Use wal project to store Write-Ahead-Log and Value-Log
  * Hash key into different indexes and value log files
  * Index and Vlog read/write will be concurrent
* Basic Put/Get/Delete/Exist operations
* Batch Put/Delete operations
* Value log compaction

<hr /><em>This discussion was created from the release <a href='https://github.com/lotusdblabs/lotusdb/releases/tag/v2.0.0'>Release v2.0.0</a>.</em>

**Comment by prr123** [comment]
congratulations!
tested code
